.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Qhat os the importance of a `.envrc` file? How does it work woth flakes here? Please briefly explain.

Reply:
This `.envrc` file is for [`direnv`](https://direnv.net/). If you're using `direnv`, you can allow this file to be loaded if you navigate into this directory. It will automatically load the default devShell for you.